.github/workflows/release_dockerhub_image.yml

```diff
@@ -82,26 +93,27 @@ jobs:
         env:
           VERBOSE: "false"
         run: breeze ci selective-check 2>> ${GITHUB_OUTPUT}
+      - name: "Determine build matrix"
+        shell: bash
+        id: determine-matrix
+        run: |
+          if [[ "${AMD_ONLY}" = "true" ]]; then
+             echo 'platform-matrix=["linux/amd64"]' >> "${GITHUB_OUTPUT}"
+          else
+                echo 'platform-matrix=["linux/amd64", "linux/arm64"]' >> "${GITHUB_OUTPUT}"
+          fi
 
-  release-images:
-    timeout-minutes: 120
-    name: "Release images: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}"
-    runs-on: ["ubuntu-22.04"]
+  build-images:
+    timeout-minutes: 50
+    # yamllint disable rule:line-length
+    name: "Build: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}, ${{ matrix.platform }}"
+    runs-on: ${{ matrix.platform == 'linux/amd64' && fromJSON('["ubuntu-22.04"]') || fromJSON('["asf-arm"]') }}
```

Comment:
Unfortunately - we cannot merge that one yet. The Arm self-hosted images we have available from INFRA do not have hypervisors and we cannot really start docker on them and build the images. We might maybe want to do it with Podman (I will try) but I'd strongly prefer to have our K8S-ARC controlled ARM runners for those.

cc: @hussein-awala

Reply:
Actually.. It turns out that we can :). Seems that public `ubuntu-22.04-arm` runners work well.